Revised remote registration docs (#31)
glencoesoftware/omero2pandas: README.md

A `variables` map usually isn't needed for simple queries. The basic condition string should automatically get converted to a meaningful type, but when this fails 
replacing tricky elements with a variable may help.

### Remote registration [Experimental - OMERO Plus Only]

For **OMERO Plus** installations which support TileDB as the OMERO.tables backend 

> it is possible to register tables in-place in a similar manner to in-place image 

Reviewer: Can you also clarify the language here? Non-remote table data is stored in OMERO's binary repository, but not within `ManagedRepository`.

# Returns the path to the created tiledb file
```

> This will convert the table into a TileDB database which will be written to `local_path`, then attempt to 
> register this to OMERO in-place. For this to work the local_path needs to be visible on the server machine 
> as well (e.g. a network drive).

Reviewer: ```suggestion
This will convert the table into a TileDB file written to `local_path`, then attempt to 
register this table to OMERO Plus in-place. For this to work the `local_path` needs to be readable via the server machine 
as well (e.g. a network drive).
```

If shared storage is mounted differently from the server's point of view, you can also supply the `remote_path` 

> parameter to declare where OMERO should find the resulting TileDB file.

Reviewer: ```suggestion
parameter to declare where OMERO Plus should find the resulting TileDB file.
```

> For example, if registering from a Windows machine with a network drive to an OMERO server on Linux:

Reviewer: ```suggestion
For example, if registering from a Windows machine with a network drive to an OMERO Plus server on Linux:
```

implies that both machines will see the file at the local path.

> Note that when a table is registered remotely it is not part of the Managed Repository 

Reviewer: ```suggestion
Note that when a table is registered remotely it is not stored within the binary repository 
```
